source/data_model/dataset/RepertoireDataset.py

```diff
@@ -46,14 +47,26 @@ def get_repertoire(self, index: int = -1, repertoire_identifier: str = "") -> Re
     def get_example_count(self):
         return len(self.repertoires)
 
+    def get_metadata_file(self):
```

Comment:
get_metadata_file() should be a fix for galaxy issues when metadata is in the same folder? If that’s the case, would it not be better to fix that when importing data vs fixing it every time in the RepertoireDataset class? :)

Reply:
updated now